### DISCUSSION TITLE
is this compatible with actix-web?

### DISCUSSION THREAD

**Discussion by Cariaga** [paritytech/jsonrpsee#1205] [Q&A] [answered]
is this compatible with actix-web?

**Comment by niklasad1** [comment, marked as answer]
Hey,

Currently, it isn't compatible with actix-web or any other web framework.

I haven't used myself that much but if it would be possible to inject the RpcModule in a actix-handler/route and just call into that RpcModule that should work without using the default HTTP/WS server in jsonrpsee.

I guess it makes sense to keep everything on a single socket if actix-web is already in your project.

**Reply by Cariaga** [reply]
i see thank you that save me a bit of headache i guess i have to jsonrpsee differently or just base on the sample.
